content/news/XXXX-XX-XX-mixxx-2-3-3-released.md

```diff
@@ -0,0 +1,50 @@
+title: Mixxx 2.3.3 released
+authors: Nikolaus Einhauser
+tags: 2.3, 2.3.3, release announcement
+comments: yes
+status: draft
+
+# Mixxx 2.3.3 Released
+
+Its that time again!  
+Almost a year after [the release of 2.3.0]({filename}/news/2021-06-28-mixxx-2-3-0-released.md)
+we're bringing you the next bugfix release [Mixxx 2.3.3](https://github.com/mixxxdj/mixxx/releases/tag/2.3.3).
+
+This release features - you guessed it - various bugfixes and improvements:
+But since we're not your run-off-the-mill mobile App, we'll get a bit into
+detail:
+
+A bunch of controller mappings have received various improvements,
+such as the Pioneer-DDJ-SB3, the NI Traktor S3, Behringer DDM4000 and Denon MC7000.
```

Comment:
```suggestion
such as the Pioneer DDJ-SB3, the NI Traktor S3, Behringer DDM4000 and Denon MC7000.
```

Maybe we can also add links to the manual pages? Just an idea.

Reply:
IMO we don't need to link